modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -138,36 +168,41 @@ function FieldBase({
 					</div>
 				)}
 
-				{((label && showLabel) ||
-					required ||
-					tooltip ||
-					repeatable) && (
-					<label
-						className={classNames({
-							'ddm-empty': !showLabel && !required,
-							'ddm-label': showLabel || required,
-						})}
-						htmlFor={id ? id : name}
-					>
-						{label && showLabel && label}
-
-						{required && (
-							<span className="reference-mark">
-								<ClayIcon symbol="asterisk" />
-							</span>
-						)}
-
-						{tooltip && (
-							<span className="ddm-tooltip">
-								<ClayIcon
-									data-tooltip-align="right"
-									symbol="question-circle-full"
-									title={tooltip}
-								/>
-							</span>
-						)}
-					</label>
-				)}
+				{showLegend &&
+					((label && showLabel) ||
+						required ||
+						tooltip ||
+						repeatable) && (
+						<legend className="lfr-ddm-legend">
+							{label && showLabel && label}
+
+							<FieldProperties
+								required={required}
+								tooltip={tooltip}
+							/>
+						</legend>
+					)}
+
+				{!showLegend &&
+					((label && showLabel) ||
+						required ||
+						tooltip ||
+						repeatable) && (
```

Comment:
@carolmariaabb, try a ternary operator here to avoid duplicate the check as is done [here](https://github.com/liferay/liferay-portal/blob/38b09127ba6c6277c9377c1d0bb78fbc07c90ae7/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Geolocation/Geolocation.es.js#L58-L80).